src/wallet/walletdb.cpp

```diff
@@ -1012,6 +1014,7 @@ std::unique_ptr<WalletDatabase> MakeDatabase(const fs::path& path, const Databas
         if (ExistsBerkeleyDatabase(path)) {
             format = DatabaseFormat::BERKELEY;
         }
+#ifdef USE_SQLITE
```

Comment:
No need for this since `ExistsSQLiteDatabase` is false.

Reply:
It is needed, as `ExistsSQLiteDatabase` is defined in `wallet/sqlite.h`.